### ISSUE TITLE
Thesis 

### ISSUE THREAD

**Issue by cbonnett** [axelbrando/Mixture-Density-Networks-for-distribution-and-uncertainty-estimation#2]
Hi Axel,

Is the thesis available yet ? 

Regards,

Christopher

**Comment by axelbrando** [comment]
Hi Christopher,

Thank you very much for your interest in the work I did.

In order to be able to add more details to the explanations and to make it easier for anyone who wanted to comment or correct written parts of the report of the Master's Thesis, my idea was to realize a web view of the final master's work in the coming days. However, I uploaded a [PDF version](https://github.com/axelbrando/Mixture-Density-Networks-for-distribution-and-uncertainty-estimation/blob/master/ABrando-MDN-MasterThesis.pdf) with some typographical errors corrected with respect to the version that the university will publish publicly (until they do not correct it).

Any contribution or idea to continue the lines of the proposed work will be very welcome.

Regards,
Axel
